### ISSUE TITLE
Check if 1080p player size does anything

### ISSUE THREAD

**Issue by SpaceOgre** [SpaceOgre/smarttv-twitch#14]
Some of the TVs support 1080p (I know high def right ;)) but the all of the resolutions used in the app is for 720p.

Investigate if there is any quality difference if player etc is set to 1080p. If yes an option in settings is required.

**Comment by Fansh1ne** [comment]
I use 720p for Twitch (tv is 1080p ofc). Using higher res 720p60, 1080p is causing some audio stuttering on my tv (1080p60 says "format not supported"). Guess it's my tv's hardware.

**Comment by SpaceOgre** [comment]
Yeah different TVs will handle it differently. If I go ahead with this it will be configurable in settings.

**Comment by SpaceOgre** [comment]
Yeah I'm not even sure if it affects playing at all or just the size of elements and stuff in the UI.

This is not connected to the quality settings but the size of the app it self.
